pkg/verifier/types/types.go

```diff
@@ -26,6 +26,7 @@ const (
 	SpecVersion      string = "0.1.0"
 	Version          string = "version"
 	Name             string = "name"
+	SpecName         string = "specName"
```

Comment:
I feel `SpecName` is specific to CR context, but for CLI users, they might have no context on it and  be confused by the name. Maybe we can just name it as `Type`.

Reply:
+1